src/content/docs/tutorials/map-data-to-ocsf.mdx

```diff
@@ -358,20 +355,30 @@ The Context attributes provide enhancing information. Most notably, the
 // === Context ===
 ocsf.metadata = {
   log_name: "conn.log",
-  logged_time: zeek._write_ts,
+  logged_time: move zeek.?_write_ts,
```

Comment:
Note: Depending on whether we merge https://github.com/tenzir/tenzir/pull/5128 this may need to be adapted.